src/parser/bison_parser.y

```diff
@@ -607,13 +609,11 @@ select_statement:
 			$$ = $2;
 			$$->withDescriptions = $1;
 		}
-	|	opt_with_clause select_with_paren set_operator select_paren_or_clause opt_order opt_limit {
-			// TODO: allow multiple unions (through linked list)
-			// TODO: capture type of set_operator
-			// TODO: might overwrite order and limit of first select here
+	|	opt_with_clause select_with_paren set_operator select_statement_in_set_operation opt_order opt_limit {
```

Comment:
What about a Query like this:
```
( SELECT * FROM A EXCEPT SELECT * FROM B ) EXCEPT SELECT * FROM C;
```
Wouldn't the "EXCEPT SELECT * FROM B " part simply get overwritten by the "EXCEPT SELECT * FROM C" part?

Reply:
True, same problem as above though. Solution will complicate some unrelated queries. Will see if this desirable

Reply:
Fixed in newest Patch